ui/DesignSystem/Primitive/Input/Text.svelte

```diff
@@ -11,7 +11,7 @@
   export let disabled = null;
   export let valid = true;
   export let validationMessage = null;
-  export let variant = "vanilla"; // vanilla | handle
+  export let variant = "vanilla"; // vanilla | handle | project
```

Comment:
These variants seem again like a tight coupling, is there a way to solve this is in a different way. For example combine general purpose components into higher level single-purpose components? Integrated on a opaque string doesn't sound too appealing.

Reply:
This got resolved by merging in the latest master.